File: app/src/main/java/com/karim/booksapp/data/models/pojos/Offer.java

package com.karim.booksapp.data.models.pojos;


import com.squareup.moshi.Json;

public class Offer {

    @Json(name = "finskyOfferType")
    public Integer finskyOfferType;
    @Json(name = "listPrice")
    public ListPrice_ listPrice;
    @Json(name = "retailPrice")
    public RetailPrice_ retailPrice;
    @Json(name = "giftable")
    public Boolean giftable;

}

File: app/src/main/java/com/karim/booksapp/data/models/pojos/Offer.kt
package com.karim.booksapp.data.models.pojos


import com.squareup.moshi.Json

class Offer {

    @Json(name = "finskyOfferType")
    var finskyOfferType: Int? = null
    @Json(name = "listPrice")
    var listPrice: ListPrice_? = null
    @Json(name = "retailPrice")
    var retailPrice: RetailPrice_? = null
    @Json(name = "giftable")
    var giftable: Boolean? = null

}
